msgpack-jackson/src/test/java/org/msgpack/jackson/dataformat/benchmark/Benchmarker.java

```diff
@@ -0,0 +1,95 @@
+//
+// MessagePack for Java
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+package org.msgpack.jackson.dataformat.benchmark;
+
+import org.apache.commons.math3.stat.StatUtils;
+import org.apache.commons.math3.stat.descriptive.moment.StandardDeviation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Benchmarker
+{
+    private final List<Benchmarkable> benchmarkableList = new ArrayList<Benchmarkable>();
+
+    public abstract static class Benchmarkable
+    {
+        private final String label;
+
+        protected Benchmarkable(String label)
+        {
+            this.label = label;
+        }
+
+        public abstract void run() throws Exception;
+    }
+
+    public void addBenchmark(Benchmarkable benchmark)
+    {
+        benchmarkableList.add(benchmark);
+    }
+
+    private static class Tuple<F, S>
+    {
+        F first;
+        S second;
+
+        public Tuple(F first, S second)
+        {
+            this.first = first;
+            this.second = second;
+        }
+    }
+
+    public void run(int count, int warmupCount)
+            throws Exception
+    {
+        List<Tuple<String, double[]>> benchmarksResults = new ArrayList<Tuple<String, double[]>>(benchmarkableList.size());
+
+        for (Benchmarkable benchmark : benchmarkableList) {
+            double[] durations = new double[count];
+
+            for (int i = 0; i < count + warmupCount; i++) {
+                if (i >= warmupCount) {
+                    System.gc();
+                }
+
+                long currentTimeNanos = System.nanoTime();
+                benchmark.run();
+
+                if (i >= warmupCount) {
+                    durations[i - warmupCount] = (System.nanoTime() - currentTimeNanos) / 1000000.0;
+                }
+            }
+            benchmarksResults.add(new Tuple<String, double[]>(benchmark.label, durations));
+        }
+
+        for (Tuple<String, double[]> benchmarkResult : benchmarksResults) {
+            printStat(benchmarkResult.first, benchmarkResult.second);
+        }
+    }
+
+    private void printStat(String label, double[] values)
+    {
+        StandardDeviation standardDeviation = new StandardDeviation();
+        System.out.println(label + ":");
+        System.out.println(String.format("  mean : %8.3f", StatUtils.mean(values)));
```

Comment:
According to the result of profiling, this method accounts for 1% of the total duration. Also, we compare the performance of Jackson and MessagePack relatively with this benchmark even if this method affects a bit.